src/components/storage/ShoppingCartTableSkeleton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Skeleton } from '@/components/ui/Skeleton';
+
+export default function ShoppingCartTableSkeleton() {
+  return (
+    <>
+      <div className='mt-4 grid grid-cols-10 gap-2'>
+        {Array.from({ length: 4 }).map((_, index) => (
+          <Skeleton className='col-span-1 h-5' key={`s_1_${index + 1}`} />
```

Comment:
we should be consistent on the skeleton keys. maybe use Skeleton index + 1. Maybe also change it on news, just so it is consistant.

Reply:
Not sure how you would like me to change this, while also not creating duplicate keys. For example `Skeleton_1`, `Skeleton_2`, and so on will make React complain if the skeletons further down on the page generate keys in the same way

Reply:
Actually try using this: https://react.dev/reference/react/useId. It is actually a hook for aria labels, but I think this is also an appropriate place to use it. Also change it on the news skeleton if it works so it is consistant since on the news skeleton we used: "Grid" + index + 1

Reply:
Applied in f1a904ec69aff1b1a08287d2589ba2704d1e14c2.